unsupported/test/cxx11_tensor_scan_cuda.cu

```diff
@@ -16,7 +16,6 @@
 #include "main.h"
 #include <unsupported/Eigen/CXX11/Tensor>
 
-
```

Comment:
undo this cosmetic change?

Reply:
hmm...will do. did not think I had modified any "cuda" file...wonder how this snuck through.